examples/ecs/generic_systems.rs

```diff
@@ -0,0 +1,24 @@
+use bevy::prelude::*;
+use bevy::ecs::component::Component;
+
+struct A;
+struct B;
+
+pub fn spawn_entities<T: Component>(mut commands: Commands,
```

Comment:
Part of me wants to suggest that we actually make this a system with two generic arguments, one for the component type and the second a const generic for which button is pressed. I wonder if that would work.

Reply:
@alice-i-cecile so for the current example basically add in a "const" argument in the function?

Also sorry I haven't been very active recently was sick this last week and haven't been able to do
a lot of coding.